app/i18n/en/gen.php

```diff
@@ -32,6 +32,7 @@
 		'purge' => 'Purge',
 		'remove' => 'Remove',
 		'rename' => 'Rename',
+		'see_url' => 'See URL',	// TODO
```

Comment:
`View URL` would be the correct form of this, but I think `Open URL` would be better.

```suggestion
		'see_url' => 'Open URL',
```

Reply:
should I rename the key too?

Reply:
At this stage definitely. :+1:

Reply:
done